src/art/FabianCristancho/style.css

```diff
@@ -0,0 +1,26 @@
+body,
+html {
+  height: 100%;
+  background-color: #485460;
+  padding: 0;
+  margin: 0;
+}
+
+.viewport {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+}
+
+/* .workspace {
```

Comment:
Same here.

Reply:
thanks for the clarification. I just deleted the commented lines, so please check it :D